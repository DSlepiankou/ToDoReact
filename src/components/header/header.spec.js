import Header from "./header";
import {render}  from '@testing-library/react';
import { Provider } from "react-redux";
import configureStore from 'redux-mock-store';

const mockStore = configureStore([]);

const testStore = mockStore({
    toDoList: {
        elements: [{}, {}, {}]
    }
})

describe('Snapshot header testing', () => {

    const headerComponent = render(
        <Provider store={testStore}>
            <Header/>
        </Provider>
    )

    it('renders properly', () => {
        expect(headerComponent).toMatchSnapshot()
    })
});