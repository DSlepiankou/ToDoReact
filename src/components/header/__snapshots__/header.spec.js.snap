// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`qwe renders properly 1`] = `ShallowWrapper {}`;
